Pod/PassportScannerController.swift

```diff
@@ -252,12 +298,55 @@ open class PassportScannerController: UIViewController, MGTesseractDelegate {
     }
     
     
+    /**
+     The frame of the ocr parsing. It is in locical pixel, relative to the vc's view.
+     The frame is used to show the area that will be scanned (if debug = true)
+     The real scan area is converted to a coordinate relative to the final image (fullhd frame)
+     */
+    @objc public func setOcrParsingRect(frame: CGRect){
+        
+        let videoFrameSize : CGSize = CGSize(width: 1080, height: 1920)
+        
+        let scale : CGFloat = UIScreen.main.scale
+        
+        let x : CGFloat = (frame.origin.x * scale * videoFrameSize.width)  / (self.view.frame.size.width  * scale)
+        let y : CGFloat = (frame.origin.y * scale * videoFrameSize.height) / (self.view.frame.size.height * scale)
+        
+        let w : CGFloat = (frame.size.width  * scale * videoFrameSize.width)  / (self.view.frame.size.width  * scale)
+        let h : CGFloat = (frame.size.height * scale * videoFrameSize.height) / (self.view.frame.size.height * scale)
+        
+        self.ocrParsingRect = CGRect(x: x, y: y, width: w, height: h)
+        
+        crop.cropSizeInPixels = Size(width: Float(ocrParsingRect.size.width), height: Float(ocrParsingRect.size.height))
+        crop.locationOfCropInPixels = Position(Float(ocrParsingRect.origin.x), Float(ocrParsingRect.origin.y), nil)
+        self.tesseract.rect = CGRect(x: 0, y: 0, width: ocrParsingRect.size.height / 2, height: ocrParsingRect.size.width / 2)
+        
+    }
+    
+    func getOcrParsingRectDebugView() -> CGRect{
+        let videoFrameSize : CGSize = CGSize(width: 1080, height: 1920)
+        
+        let h : CGFloat = (self.ocrParsingRect.size.height / videoFrameSize.height) * renderView.frame.size.height
+        let w : CGFloat = (self.ocrParsingRect.size.width / videoFrameSize.width) * renderView.frame.size.width
+        let x : CGFloat = ((self.ocrParsingRect.origin.x / videoFrameSize.width) * renderView.frame.size.width)
+        let y : CGFloat = ((self.ocrParsingRect.origin.y / videoFrameSize.height) * renderView.frame.size.height)
+        
+        return CGRect(x: x, y: y, width: w, height: h)
+    }
+    
+    
     @objc public func startScan() {
+        self.setup()
```

Comment:
What is the point of calling `setup` in the `viewDidLoad` and subsequently also here? Same about the `initCamera`. There is so much redundant code here...